CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.3.0 (2016-2-19)
```

Comment:
Can you try and figure out why these are showing up as a diff? It looks like GitHub thinks two of _my_ commits are new...which worries me a little.